### ISSUE TITLE
Skipping docker build for UI

### ISSUE THREAD

**Issue by stlouisweb** [capitalone/Hygieia#359]
I'm trying to run Hygieia via Docker-Compose on my mac.

I seem to have made it through the api portion of the install alright, but when I try to build the UI container mvn goes through its tasks, and seems to move along fine until it hits the docker build task

This leads me to believe that npm, bower and gulp do their thing, not seeing any errors there. So why is the docker build being skipped? I'm not getting any errors reported here either.

```
JohnDoes-MacBook-Pro:hygieia johndoe$ mvn clean package -pl UI docker:build
[INFO] Scanning for projects...
[INFO]                                                                         
[INFO] ------------------------------------------------------------------------
[INFO] Building com.capitalone.dashboard:UI 2.0.0-SNAPSHOT
[INFO] ------------------------------------------------------------------------
[INFO] 
[INFO] --- maven-clean-plugin:2.5:clean (default-clean) @ UI ---
[INFO] 
[INFO] >>> maven-pmd-plugin:3.5:check (default) > :pmd @ UI >>>
[INFO] 
[INFO] --- maven-pmd-plugin:3.5:pmd (pmd) @ UI ---
[INFO] 
[INFO] <<< maven-pmd-plugin:3.5:check (default) < :pmd @ UI <<<
[INFO] 
[INFO] --- maven-pmd-plugin:3.5:check (default) @ UI ---
[INFO] 
[INFO] --- frontend-maven-plugin:0.0.27:install-node-and-npm (install node and npm) @ UI ---
[INFO] Found proxies: []
[INFO] Node v5.4.1 is already installed.
[INFO] Found NPM version 2.14.15
[INFO] 
[INFO] --- frontend-maven-plugin:0.0.27:npm (npm install) @ UI ---
[INFO] Found proxies: []
[INFO] Running 'npm install --color=false' in /Users/johndoe/somedir/hygieia/UI
[INFO] 
[INFO] --- frontend-maven-plugin:0.0.27:bower (bower install) @ UI ---
[INFO] Found proxies: []
[INFO] Running 'bower install' in /Users/johndoe/somedir/hygieia/UI
[INFO] 
[INFO] --- frontend-maven-plugin:0.0.27:gulp (gulp build) @ UI ---
[INFO] Running 'gulp.js build --no-color' in /Users/johndoe/somedir/hygieia/UI
[INFO] [11:41:45] Using gulpfile ~/somedir/hygieia/UI/gulpfile.js
[INFO] [11:41:45] Starting 'build'...
[INFO] [11:41:45] Starting 'clean'...
[INFO] [11:41:45] Finished 'clean' after 75 ms
[INFO] [11:41:45] Starting 'assets'...
[INFO] [11:41:45] Starting 'themes'...
[INFO] [11:41:45] Starting 'fonts'...
[INFO] [11:41:45] Starting 'js'...
[INFO] [11:41:45] Starting 'views'...
[INFO] [11:41:45] Starting 'test-data'...
[INFO] [11:41:50] Finished 'test-data' after 5.43 s
[INFO] [11:41:52] Finished 'themes' after 6.93 s
[INFO] [11:41:52] Finished 'assets' after 6.94 s
[INFO] [11:41:52] Finished 'views' after 7.02 s
[INFO] [11:41:52] Finished 'js' after 7.12 s
[INFO] [11:41:52] Finished 'fonts' after 7.44 s
[INFO] [11:41:52] Starting 'html'...
[INFO] [11:41:52] gulp-inject 1 files into index.html.
[INFO] [11:41:52] gulp-inject 89 files into index.html.
[INFO] [11:41:52] Finished 'html' after 185 ms
[INFO] [11:41:52] Finished 'build' after 7.73 s
[INFO] 
[INFO] --- docker-maven-plugin:0.3.3:build (default-cli) @ UI ---
[INFO] Skipping docker build
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 25.425 s
[INFO] Finished at: 2016-02-22T11:41:54-06:00
[INFO] Final Memory: 34M/333M
[INFO] ------------------------------------------------------------------------
JohnDoes-MacBook-Pro:hygieia johndoe$ 
```

**Comment by jimzucker** [comment]
I am taking a look at this...


On Feb 22, 2016, at 9:52 AM, Jeremy Plack <notifications@github.com<mailto:notifications@github.com>> wrote:


I'm trying to run Hygieia via Docker-Compose on my mac.

I seem to have made it through the api portion of the install alright, but when I try to build the UI container mvn goes through its tasks, and seems to move along fine until it hits the docker build task

This leads me to believe that npm, bower and gulp do their thing, not seeing any errors there. So why is the docker build being skipped? I'm not getting any errors reported here either.

JohnDoes-MacBook-Pro:hygieia johndoe$ mvn clean package -pl UI docker:build
[INFO] Scanning for projects...
[INFO]
[INFO] ------------------------------------------------------------------------
[INFO] Building com.capitalone.dashboard:UI 2.0.0-SNAPSHOT
[INFO] ------------------------------------------------------------------------
[INFO]
[INFO] --- maven-clean-plugin:2.5:clean (default-clean) @ UI ---
[INFO]
[INFO] >>> maven-pmd-plugin:3.5:check (default) > :pmd @ UI >>>
[INFO]
[INFO] --- maven-pmd-plugin:3.5:pmd (pmd) @ UI ---
[INFO]
[INFO] <<< maven-pmd-plugin:3.5:check (default) < :pmd @ UI <<<
[INFO]
[INFO] --- maven-pmd-plugin:3.5:check (default) @ UI ---
[INFO]
[INFO] --- frontend-maven-plugin:0.0.27:install-node-and-npm (install node and npm) @ UI ---
[INFO] Found proxies: []
[INFO] Node v5.4.1 is already installed.
[INFO] Found NPM version 2.14.15
[INFO]
[INFO] --- frontend-maven-plugin:0.0.27:npm (npm install) @ UI ---
[INFO] Found proxies: []
[INFO] Running 'npm install --color=false' in /Users/johndoe/somedir/hygieia/UI
[INFO]
[INFO] --- frontend-maven-plugin:0.0.27:bower (bower install) @ UI ---
[INFO] Found proxies: []
[INFO] Running 'bower install' in /Users/johndoe/somedir/hygieia/UI
[INFO]
[INFO] --- frontend-maven-plugin:0.0.27:gulp (gulp build) @ UI ---
[INFO] Running 'gulp.js build --no-color' in /Users/johndoe/somedir/hygieia/UI
[INFO] [11:41:45] Using gulpfile ~/somedir/hygieia/UI/gulpfile.js
[INFO] [11:41:45] Starting 'build'...
[INFO] [11:41:45] Starting 'clean'...
[INFO] [11:41:45] Finished 'clean' after 75 ms
[INFO] [11:41:45] Starting 'assets'...
[INFO] [11:41:45] Starting 'themes'...
[INFO] [11:41:45] Starting 'fonts'...
[INFO] [11:41:45] Starting 'js'...
[INFO] [11:41:45] Starting 'views'...
[INFO] [11:41:45] Starting 'test-data'...
[INFO] [11:41:50] Finished 'test-data' after 5.43 s
[INFO] [11:41:52] Finished 'themes' after 6.93 s
[INFO] [11:41:52] Finished 'assets' after 6.94 s
[INFO] [11:41:52] Finished 'views' after 7.02 s
[INFO] [11:41:52] Finished 'js' after 7.12 s
[INFO] [11:41:52] Finished 'fonts' after 7.44 s
[INFO] [11:41:52] Starting 'html'...
[INFO] [11:41:52] gulp-inject 1 files into index.html.
[INFO] [11:41:52] gulp-inject 89 files into index.html.
[INFO] [11:41:52] Finished 'html' after 185 ms
[INFO] [11:41:52] Finished 'build' after 7.73 s
[INFO]
[INFO] --- docker-maven-plugin:0.3.3:build (default-cli) @ UI ---
[INFO] Skipping docker build
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 25.425 s
[INFO] Finished at: 2016-02-22T11:41:54-06:00
[INFO] Final Memory: 34M/333M
[INFO] ------------------------------------------------------------------------
JohnDoes-MacBook-Pro:hygieia johndoe$

**Comment by jimzucker** [comment]
@stlouisweb take a look at the commit I made on this ticket, if you pull that change it will fix your pom.xml

**Comment by stlouisweb** [comment]
that was quick! I'm testing it out now. I'll let you know how it goes in a few.

**Comment by stlouisweb** [comment]
it works, no more `Skipping docker build`

**Comment by stlouisweb** [comment]
figured maybe leave this open until your change gets merged?

**Comment by jimzucker** [comment]
merged 4f20dca - (upstream/master, origin/master, origin/HEAD, master) Merge pull request #341 from jimzucker/eclipseCleanup

**Comment by tabladrum** [comment]
closed now with @jimzucker comment above.
